kview/src/main/java/dev/ikm/komet/kview/events/EventTopics.java

```diff
@@ -18,4 +18,5 @@
 public enum EventTopics {
     JOURNAL_TOPIC,
     USER_TOPIC,
+    SAVE_PATTERN_TOPIC,
```

Comment:
(optional) we didn't use the JOURNAL_TOPIC?  I don't know if LIDR or Concept would need to listen to it but they could listen on the JOURNAL_TOPIC...?